honeybee_radiance/properties/model.py

```diff
@@ -310,6 +467,72 @@ def check_duplicate_modifier_set_identifiers(self, raise_exception=True):
             return False
         return True
 
+    def check_duplicate_sensor_grid_identifiers(self, raise_exception=True):
```

Comment:
I had to resolve a similar issue before and this was the best option that I found online.

```py
dup = [t for t, c in collections.Counter(grid.identifier for grid in self.sensor_grids).items() if c > 1]
```

Reply:
I'm not totally convinced it's the most readable or beautiful but it's the shortest.  If we decide that we like it, there are some ~30 methods that should be changed across honeybee-core, dragonfly-core and the extensions.

Reply:
In that case, we should probably make one method in honeybee-core and have every other library to use it. It is not critical of course as it will work as is.

Reply:
Ok. I opened an issue for it https://github.com/ladybug-tools/honeybee-core/issues/198